packages/bsky/src/api/app/bsky/actor/getSuggestions.ts

```diff
@@ -110,6 +112,24 @@ const presentation = (state: HydrationState) => {
   return { actors: suggestedActors, cursor }
 }
 
+const parseCursor = (cursor?: string): number[] => {
+  if (!cursor) {
+    return []
+  }
+  try {
+    return cursor.split(':').map((id) => parseInt(id))
```

Comment:
If someone is mucking around with the API manually, it's likely you'll get some `NaN`s out of this.

Reply:
good call - i'll throw a filter in 👍